dmutils/direct_plus_client.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import requests
+
+
+class DirectPlusClient(object):
```

Comment:
We have a mixture of naming here - `DirectPlus` and `dnb`. Worth having a brief docstring explaining what they are and what this is for?

Reply:
Have updated the references to direct_plus and added a docstring.

direct_plus is more specific as this isn't an interface to dun and bradstreet bu _only_ their direct plus api